apps/i18n/eval/en_us.json

```diff
@@ -6,6 +6,7 @@
   "ellipseBlockTitle": "ellipse (width, height, style, color)",
   "emptyFunctionalBlock": "You have a block with an unfilled input.",
   "extraTopBlocks": "You have unattached blocks. Did you mean to attach these to the \"display\" block?",
+  "infiniteRecursionError": "Infinite recursion detected in your code.",
```

Comment:
More seriously, perhaps: "Your function is calling itself.  We have stopped it, otherwise it would have continued calling itself forever."